beam/README.md

```diff
@@ -22,7 +24,7 @@ You will generate two categories of artifacts for this initialization action:
 When building manually, substitute the following terms into build commands. In
 bash, set environment variables using `export <term>=<value>`.
 
-| `BEAM_JOB_SERVICE_DESTINATION` | A Cloud Storage directory path accessible by both the build machine and the cluster to be created. |
+| `beam_DESTINATION` | A Cloud Storage directory path accessible by both the build machine and the cluster to be created. |
```

Comment:
`beam_DESTINATION` -> `BEAM_DESTINATION`

Looks like this is used in several places; I assume this was just a bad substitution.

Reply:
Yep.  Good catch.  Thanks.  Fixed.